web/packages/extension/assets/_locales/en/messages.json

```diff
@@ -21,7 +21,7 @@
         "message": "Log level"
     },
     "settings_max_execution_duration": {
-        "message": "How long can ActionScript execute for before being forcefully stopped (in seconds)"
+        "message": "Maximum allowed ActionScript execution duration (in seconds)"
     },
```

Comment:
Would it make sense to add translation strings for both `preferred_renderer` and `force_renderer`? I think it might have been forgotten in https://github.com/ruffle-rs/ruffle/pull/9904 (I forgot it myself in at least one PR until I got reminded of it in a different PR). Unless it was omitted on purpose in https://github.com/ruffle-rs/ruffle/pull/9904 .

From what I remember of the code, it is only necessary to make entries here, since the ID value and `for` value in the HTML is used for the translation key, so the entry keys would be `settings_preferred_renderer` and `settings_force_renderer` .

Reply:
I didn't know about adding translation strings, so yeah, that was an oversight on my part.